packages/node_modules/@ciscospark/plugin-messages/src/messages.js

```diff
@@ -28,6 +41,72 @@ import {isArray} from 'lodash';
  * @class
  */
 const Messages = SparkPlugin.extend({
+  /**
+   * Initializer
+   * @param  {...any} args
+   * @returns {undefined}
+   */
+  initialize(...args) {
+    Reflect.apply(SparkPlugin.prototype.initialize, this, args);
+
+    // Listen to new activities from Mercury.
+    this.listenTo(this.spark.internal.mercury, 'event:conversation.activity',
+      (event) => this.onConversationActivityEvent(event));
```

Comment:
I feel like this code should go inside the listen method.

Reply:
And you could optionally call listen automatically to prevent an end user from having to do it?

Reply:
That's what I was going back and forth on. I can hook up mercury by default, but wasn't sure if I should.

Reply:
@andrewholsted I moved listening into one function.

Reply:
@mattnorris, when we are all said and done with this work, ideally we'll also have events coming out of the memberships and rooms module as well.    We could add a listen() method to each of those modules as well, but are there inefficiencies on having 3 different plugins all listening for all the conversation activity events on the same socket?

If not that is certainly the most readable way to do this implementation.  If so it seems like we'd still like to have the developer do something proactive to indicate that they want message (or membership, or room) events, but to have a common internal socket plugin that routes the appropapriate conversation activities to the appropriate module handler.

Reply:
@jpjpjp good point; they will all listen to the same socket, so there won't be inefficiencies.